drltrace/drltrace.config

```diff
@@ -0,0 +1,8 @@
+; The syntax of this config file is simple. To add new function, it's enough to specify
+; each argument of the function, function name itself and return type separated by commas.
```

Comment:
I'm talking here only about fundamental issues with the proposed syntax, mainly the choice of using comma as a separator.  Adding annotations, as you mention for output params, later makes perfect sense.